index.html

```diff
@@ -118,7 +118,7 @@ <h2 data-ccs-colors="invert">neutral</h2>
   </main>
 
   <footer>
-    <p>Copyright © 2019 <a href="https://miriam.codes/">Miriam Suzanne</a></p>
+    <p>Copyright © 2019–2020 <a href="https://oddbird.net/">OddBird</a></p>
```

Comment:
@mirisuzanne I don't actually care who holds the copyright, but think this should match the LICENSE.md file.

Reply:
I agree - and OddBird is better for it